<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortDialog</class>
 <widget class="QDialog" name="PortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/QTtest/QT图片/photo-1492011221367-f47e3ccd77a0.jpg);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>260</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="sureBtn">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitBtn">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>243</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>239</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>串口选择</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="portNameComboBox">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>波特率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>数据位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>奇偶校验位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>串口操作</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="baudComboBox">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>61</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>119</width>
              <height>61</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>自定义</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1382400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>921600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>460800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>230400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>76800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>43000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stopBitComboBox">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>61</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>119</width>
              <height>61</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dataBitComboBox">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>119</width>
              <height>71</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="checkBitComboBox">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>119</width>
              <height>51</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>奇校验</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>偶校验</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="openClosePushButton">
          <property name="text">
           <string>打开串口</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../QTtest/图片/微信图片_20191110172409.jpg</normaloff>../QTtest/图片/微信图片_20191110172409.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
